pkg/controller/plan/kubevirt.go

```diff
@@ -850,6 +852,40 @@ func (r *KubeVirt) GetGuestConversionPod(vm *plan.VMStatus) (pod *core.Pod, err
 	return
 }
 
+func (r *KubeVirt) GetVirtV2VConvertedVMConfig(vm *plan.VMStatus, pod *core.Pod, step *plan.Step) (err error) {
+	if pod.Status.PodIP == "" {
+		//we need the IP for fetching the configuration of the convered VM.
+		return
+	}
+
+	url := fmt.Sprintf("http://%s:8080/xml", pod.Status.PodIP)
+
+	resp, err := http.Get(url)
+	if err != nil {
+		if strings.Contains(err.Error(), "connection refused") {
```

Comment:
I believe you can check if err is ErrUnexpectedEOF directly or something similar

Reply:
Its not catching it as this error not as `ErrUnexpectedEOF` and not ad ``EOF`, leaving it as is.

Reply:
I'm not sure why I said EOF here, I think I meant to comment this on L#882, this is connection refused

here this should work: https://go.dev/play/p/CvgfnU8QdQ5

Reply:
haha ok, this one I've changed already:) I thought you were talking about the error at the end of the func.